src/views/prize-earned/prize-earned.vue

```diff
@@ -0,0 +1,67 @@
+<style lang="scss">
+@import './prize-earned.scss';
+</style>
+
+<template>
+  <div>
+    <h1>Prize unlocked!</h1>
+
+    <div v-if="visitCount === 4">
+      <TrophyBanner class="trophy-banner" :green="true" />
+      <img class="trophy" src="../../images/prizes/trophy-green.svg" alt="Green trophy" />
+    </div>
+    <div v-else-if="visitCount === 8">
+      <TrophyBanner class="trophy-banner" :purple="true" />
+      <img class="trophy" src="../../images/prizes/trophy-purple.svg" alt="Purple trophy" />
+    </div>
+    <div v-else-if="visitCount === 12">
+      <TrophyBanner class="trophy-banner" :orange="true" />
+      <img class="trophy" src="../../images/prizes/trophy-orange.svg" alt="Orange trophy" />
+    </div>
+
+    <hr class="top-line">
+    <div class="parent">
+      <div class="prize-details-container">
+        <img class="checkmark" src="../../images/checkmark.svg" alt="checkmark" />
+        <div class="prize-details">
+          <h3>Visit {{visitCount}} startups</h3>
+          <div>Win a badass prize yo!</div>
+        </div>
+      </div>
+    </div>
+    <hr>
+
+    <div class="prize-details-paragraph">
+      You've earned a prize for visiting {{visitCount}} crawl locations.
+      We've sent an email to {{personEmail}} with instructions to claim your prize.
```

Comment:
This may change based on the lottery stuff -- just need to note it in a ticket

Reply:
done! https://github.com/Skookum/stumbl-2019/projects/4#card-24048431